src/utils/scheduler.ts

```diff
@@ -102,4 +106,10 @@ class Scheduler {
 
 const scheduler = new Scheduler();
 
-export { scheduler };
+const updateCount = (n: number): void => {
+  !watching && n > 0 && scheduler.start();
+  watching += n;
+  !watching && scheduler.stop();
+}
```

Comment:
Hey @kei-ito could you move the watch count out of the scheduler. It doesn't have much relevance being here.

Reply:
OK, I'll try it. But the fix won't be straightforward because`scheduler` and `updateCount ` depend on each other.

Reply:
Hmm, ok, yeah I see what you're saying. Probably could keep this for now then and look at fixing this in  another update. Unless you've already figured something out of course 🙂

Reply:
I agree with you. As you saw, this PR just replaces some lines. It's good to leave it for simplicity.